### PR TITLE
[DSY-937] Create avatar component

### PR DESCRIPTION
**Pull request by penelopearaujo**
# Description

- Create `Avatar` component

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Internal changes

### CODE REVIEW

**Review by vitorpirestw — APPROVED** [review]
👍

File: Sources/Public/Components/Avatar/NatAvatar.swift [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+            imageView.isHidden = false
+            label.isHidden = true
+            defaultIconView.isHidden = true
+            iconView.isHidden = false
+        } else {
+            defaultFallback(icon: fallBackIcon)
+        }
+    }
+
+    public func configure(icon: String?) {
```
Comment (penelopearaujo) [inline Sources/Public/Components/Avatar/NatAvatar.swift:184]: Lembra de documentar esses novos configures :)

File: Sources/Public/Components/Avatar/NatAvatar.swift [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+    }()
+
+    internal let size: Size
+    internal var type: Types
+    internal var fallBackIcon: String?
+
+
+    // MARK: - Inits
+
+
```
Comment (penelopearaujo) [inline Sources/Public/Components/Avatar/NatAvatar.swift:86]: Fica melhor se deixar só uma linha vazia entre as linhas com código 🤔

**Comment by penelopearaujo** [comment]
Thanks!

**Pull request merged**